frontend/public/components/routes.tsx

```diff
@@ -208,16 +208,23 @@ const TLSSettings: React.FC<TLSSettingsProps> = ({ route }) => {
         label={t('network-route~Termination type')}
         obj={route}
         path="spec.tls.termination"
+        dataTest="termination-type"
```

Comment:
Instead of adding an identifier for every details item, I'm wondering if we can check something like `co-m-pane__details th` to make sure *all* the labels are translated. It's less code, and we cover new items that aren't manually added to the test. Or we can add a common `data-` attribute into `DetailsItem` for the label.

Reply:
At this time, not all labels are translated on all detail pages.  

_"...Or we can add a common data- attribute into DetailsItem for the label"_
That's what I did [here](https://github.com/openshift/console/pull/6983/files#diff-631a7e4e0f7e94fa6334e256bfae9262ddff281897e4b386eca899e30bd0afaeR72), unless I'm misunderstanding what you are asking about?

Reply:
Fixed, testing for all Detail Item Labels and introduced 'data-i18n-translated` attribute.